src/main/java/net/syscon/elite/api/model/LegalStatusCalc.java

```diff
@@ -0,0 +1,86 @@
+package net.syscon.elite.api.model;
+
+import lombok.Getter;
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.List;
+
+public class LegalStatusCalc {
+
+    @Getter
+    public enum LegalStatus {
+        RECALL("Recall"),
+        DEAD("Dead"),
+        INDETERMINATE_SENTENCE("Indeterminate Sentence"),
+        SENTENCED("Sentenced"),
+        CONVICTED_UNSENTENCED("Convicted Unsentenced"),
+        CIVIL_PRISONER("Civil Prisoner"),
+        IMMIGRATION_DETAINEE("Immigration Detainee"),
+        REMAND("Remand"),
+        UNKNOWN("Unknown"),
+        OTHER("Other");
+
+        private final String desc;
+
+        LegalStatus(String desc) {
+            this.desc = desc;
+        }
+    }
+    
+    private final static List<String> RECALL_STATUS_CODES = List.of("14FTR_ORA","14FTRHDC_ORA","CUR_ORA","FTR/08","FTR_HDC","FTR_HDC_ORA","FTR_ORA","FTR_SCH15","FTRSCH15_ORA","HDR_ORA","LR","LR_ALP","LR_ALP_LASPO","LR_DLP","LR_DPP","LR_EPP","LR_ES","LR_HDC","LR_IPP","LR_LASPO_AR","LR_LASPO_DR","LR_LIFE","LR_MLP","LR_ORA","LR_SEC236A","LR_SEC91_ORA","LR_YOI","LR_YOI_ORA"); 
+
+    public static LegalStatus getLegalStatus(final String bandCode, final String mainLegalStatusCode) {
+
+        if (StringUtils.isBlank(bandCode) || !StringUtils.isNumeric(bandCode)) {
+            return null;
```

Comment:
Why not "unknown"?

Reply:
Yes was toying with that. - That could be in the next PR when I tighten up the Recall calc